src/main/webapp/downstreamDisabled_ja.html

```diff
@@ -1,3 +1,3 @@
 <div>
-  このオプションをチェックすると、下流プロジェクトのジョブをスケジュールしません。
+  このオプションをチェックすると、下流プロジェクトのジョブをスケジュールします。
```

Comment:
Note: 924c274a497d76ac4958ddebad30759f2cd6a5b6
Other translations should have the same problem...